docs/contributors_guide/maintenance/ruby_gem_release_procedures/Release_GitHub_Deploy_RubyGems_org.rst

```diff
@@ -23,15 +23,13 @@ To create the releases from an annotated tag:
 
       git clone git@github.com:simp/rubygem-simp-rake-helpers.git
       cd rubygem-simp-rake-helpers
-      git checkout master # this step isn't needed for master branch
+      git checkout BRANCH # this step isn't needed for master branch
 
 #. Generate the changelog content
 
-   * Manually extract the changelog content from the ``CHANGELOG.md``,
-     file and write into a file.  In this example, the written file
-     will be ``foo``.
+   * ``rake changelog_annotation > foo``
```

Comment:
@trevor-vaughan Have you verified this actually works for components that are not puppet modules?

Reply:
Hmm...good point. We should probably fix the code to go dumpster diving into the `build` directory. Having two sets of manual instructions, including formatting, will probably cause issues.